CHANGELOG.md

```diff
@@ -13,10 +13,29 @@ The REST API versions in the table are just for your information as the values a
 
 ## Unreleased/planned
 
-- We are working on arithmetic measures support.
-- We are working on the option to add the secondary axis into certain visualizations (line chart, column chart, bar chart).
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major version.
 
+## 6.1.0
+
+December 17, 2018
+
+### Added
+
+- Support dual axes for Column, Bar and Line chart ([commit]
+(https://github.com/gooddata/gooddata-react-components/commit/0bc35208b098835d5e86fbe1372f120afa7d5e80)).
+- Color mapping ([commit](https://github.com/gooddata/gooddata-react-components/commit/58ad5e43b1d0f262ffd73bec8061673091409a21)).
+- Arithmetic measures.
+- Drilling using function predicates ([commit](https://github.com/gooddata/gooddata-react-components/commit/e5b55540fb8434662300dfbda95a427adf8096de)).
+
+### Changed
+
+- Translation metadata ([commit](https://github.com/gooddata/gooddata-react-components/commit/b2f38bda72e08761bf984676c73f175c8c04b410)).
+- Discarded indigo theme ([commit](https://github.com/gooddata/gooddata-react-components/commit/09e3324785e9e6fe98f3a18b5f6cd265a778625c)).
+- Fixed bug where treemap didn't assign colors properly by value ([commit]
+(https://github.com/gooddata/gooddata-react-components/commit/4e1266c7d7a98d94c9159de28e3684d466126b90)).
+- If Attribute Filter is defined by identifier then onApply parameter will have this identifier as id property value. Otherwise when Attribute Filter
+is defined by URI, the bahivour is as default ([commit])(https://github.com/gooddata/gooddata-react-components/commit/cf9244719d172eb9e16385c513000ba3ee8a50e1)).
+
```

Comment:
- Added translation metadata
- Discarded the indigo theme
- Fixed the bug where a treemap did not assign colors properly by value
- If an attribute filter is defined by the identifier, then the onApply parameter will have this identifier as the value of the ID property. Otherwise, when the attribute filter is defined by the URI, the behavior does not change.